### ISSUE TITLE
[Question] How to log Confusion Matrix in traditional representation ?

### ISSUE THREAD

**Issue by vfdev-5** [allegroai/clearml#165]
Hi,

I'm doing the following:
```python
cm = torch.eye(5, 5)
logger.report_confusion_matrix(title="Confusion Matrix", series="CM", matrix=cm.numpy(), iteration=0, xlabels=target_names[:5], ylabels=target_names[:5])
```
and I get the following image on the server.
![newplot](https://user-images.githubusercontent.com/2459423/87100060-b0a28c80-c24b-11ea-9a32-ee0630527fa2.png)

Graphical representation looks correct, but traditionnally, I'd like to see it like that:
![newplot(1)](https://user-images.githubusercontent.com/2459423/87100284-4a6a3980-c24c-11ea-986a-f30e4df206f5.png)

by doing
```
logger.report_confusion_matrix(
    title="Confusion Matrix", series="CM", matrix=cm.numpy()[::-1, :], iteration=3, xlabels=target_names[:5], ylabels=target_names[:5][::-1]
)
```

Is there is any other way to log CM without manually inversing y axis ? Thanks

**Comment by bmartinn** [comment]
Hi @vfdev-5 , 
I think @WillNichols726 mentioned the same issue [here](https://github.com/allegroai/trains/issues/81#issuecomment-583566250)
Surprisingly enough I took a second look and it looks as if this is now supported by plotly, which means we can support it without transposing manually the data.
The new interface will look something like:
```python
    cm = torch.eye(5, 5)
    logger.report_confusion_matrix(
        title="Confusion Matrix", series="CM", matrix=cm.numpy(), iteration=0, yaxis_reversed=True,)
```
And will produce a confusion matrix where the 0,0 is at the top left corner (not the bottom left corner).

WDTY?

BTW:
@WillNichols726, I double checked again, and unfortunately adding the actual values on the matrix, means sending the entire value list twice, which might be a bit much...

**Comment by vfdev-5** [comment]
@bmartinn with 0.15.2rc0 adding `yaxis_reversed`, it produces 
```
TypeError: report_confusion_matrix() got an unexpected keyword argument 'yaxis_reversed'
```

**Comment by bmartinn** [comment]
@vfdev-5 good news, the new interface was pushed, you can test it with:
```bash
pip install git+https://github.com/allegroai/trains.git
```

**Comment by allegroai-git** [comment]
Closing this as the interface is available since ClearML/Trains v0.16.0
